mlos_bench/mlos_bench/environments/local/local_env.py

```diff
@@ -204,25 +209,26 @@ def status(self) -> Tuple[Status, List[Tuple[datetime, str, Any]]]:
 
         assert self._temp_dir is not None
         try:
+            fname = self._config_loader_service.resolve_path(
+                self._read_telemetry_file, extra_paths=[self._temp_dir])
+
             # FIXME: We should not be assuming that the only output file type is a CSV.
-            data: pandas.DataFrame = pandas.read_csv(
-                self._config_loader_service.resolve_path(
-                    self._read_telemetry_file, extra_paths=[self._temp_dir]),
-                parse_dates=[0],
-            )
+            data: pandas.DataFrame = pandas.read_csv(fname, index_col=False, parse_dates=[0])
+
+            if sys.platform == 'win32':
+                data.rename(str.rstrip, axis='columns', inplace=True)
+
+            col_names = ["timestamp", "metric", "value"]
+            if len(data.columns) != len(col_names):
+                raise ValueError('Telemetry data must have columns {col_names}')
+            elif list(data.columns) != col_names:
+                # Assume no header - this is ok for telemetry data.
```

Comment:
How will this respond if there are headers, but with different names?

Reply:
it should replace the names with those in `col_names` - but yes, it's a good idea to add a unit test for it